test/github-issues/10140/issue-10140.ts

```diff
@@ -0,0 +1,47 @@
+import "reflect-metadata"
+import {
+    createTestingConnections,
+    closeTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+import { DataSource } from "../../../src"
+import { expect } from "chai"
+import { Photo } from "./entity/Photo"
+
+describe("github issues > #10140 timestamp type column doesn't work when using sqlite", () => {
+    let dataSources: DataSource[]
+    before(
+        async () =>
+            (dataSources = await createTestingConnections({
+                entities: [__dirname + "/entity/*{.js,.ts}"],
+                schemaCreate: true,
+                dropSchema: true,
+            })),
```

Comment:
It's not necessary to load the other drivers.
```suggestion
                enabledDrivers: ["sqlite"],
            })),
```